File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/sistemas/syreporte/dominio/dtosyreporte.ts
import { ParametroPaginacionGenerico } from '../../../shared/dominio/ParametroPaginacionGenerico';

export class DtoSyReporte {

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;

    aplicacionCodigo: string;
    aplicacionDescripcion: string;
    reporteCodigo: string;
    nombre: string;
    tipoReporte: string;
    estado: string;
}

File: Administrativo/spring-erp-servicio/net/royal/spring/sistema/dominio/dto/DtoSyReporte.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.sistema.dominio.dto
{
    public class DtoSyReporte
    {
        public String AplicacionCodigo { get; set; }
        public String AplicacionDescripcion { get; set; }
        public String ReporteCodigo { get; set; }
        public String Nombre { get; set; }
        public String TipoReporte { get; set; }
        public String Estado { get; set; }

        public ParametroPaginacionGenerico paginacion { get; set; }

        public DtoSyReporte()
        {
            this.paginacion = new ParametroPaginacionGenerico();
        }
    }
}
